### DISCUSSION TITLE
ViewContentList Permission

### DISCUSSION THREAD

**Discussion by luis3199correia** [OrchardCMS/OrchardCore#8798] [Q&A]
I looked for permissions to allow only to see the content created and I found [this,](https://github.com/OrchardCMS/OrchardCore/issues/7779) and I wanted to know if this permission that you are going to add will be for all the content or just for those that are published?

**Comment by Skrypt** [comment]
No one has worked on this yet so the specs still need to be defined. But I can assume that it would lock the content item from being modified after it is published. And this would be optional by content type.

**Reply by luis3199correia** [reply]
Still no one has worked on this?

**Reply by Skrypt** [reply]
Nothing done concerning locking published content items.

**Reply by luis3199correia** [reply]
@Skrypt There is a forecast of when they will work on it?
